resources/public/test.html

```diff
@@ -0,0 +1,8 @@
+<html>
```

Comment:
Since you are using the extra main feature, is there a reason to not use the index file generated by Figwheel?

>At this point you would normally need to create a host page for this new main file but you don’t need to do that because you can visit the /figwheel-extra-main/tests endpoint on your Figwheel server.

https://figwheel.org/docs/extra_mains.html

Reply:
Thanks for the comment!

AFAICT cljs-test-display requires an index.html that contains a container element with an id

https://github.com/bhauman/cljs-test-display#html-host-file

In short without test.html it didn't work cc: @bhauman